File: golang/sdk.go
package AntiSougou

import (
	"encoding/json"
	"fmt"
	"io/ioutil"
	"net/http"
)

var (
	Link string
)

const(
	BotToken string = ""
)

type Group struct {
	Ok     bool `json:"ok"`
	Result struct {
		User struct {
			Id           int    `json:"id"`
			IsBot        bool   `json:"is_bot"`
			FirstName    string `json:"first_name"`
			Username     string `json:"username"`
			LanguageCode string `json:"language_code"`
		} `json:"user"`
		Status string `json:"status"`
	} `json:"result"`
}

func Captcha(chatID int) (int,error) {
	Link = fmt.Sprintf("https://api.telegram.org/bot%s/getChatMember?chat_id=@Sougou111&user_id=%v", BotToken, chatID)
	response, err := http.Get(Link)
	if err != nil {
		return 0, err
	}
	defer response.Body.Close()
	Body, err := ioutil.ReadAll(response.Body)
	if err != nil {
		return 0, err
	}
	var JSON Group
	err = json.Unmarshal(Body, &JSON)
	if err != nil {
		return 0, err
	}
	if JSON.Ok != true {
		return 0, nil
	}
	if JSON.Result.Status == "left" || JSON.Result.Status == "kicked" {
		return 1, nil
	}
	return 0, nil
}

File: php/sdk.php
<?php
	
	/**
		You need to modify the following parameters
			::Bot Token::
	 **/
	 
class AntiSougouSGKBot{
	
    public static function Captcha(string $chatid): int
    {
        
        $Links = 'https://api.telegram.org/bot' . '::Bot Token::' . '/' . 'getChatMember?chat_id=@Sougou111&user_id=' . $chatid;
		
		//$Links = sprintf('https://api.telegram.org/bot%s/getChatMember?chat_id=@Sougou111&user_id=%s', '::Bot Token::', $chatid);
		
        $ch           = curl_init();
        curl_setopt($ch, CURLOPT_URL, $Links);
        curl_setopt($ch, CURLOPT_HEADER, 0);
        curl_setopt($ch, CURLOPT_NOSIGNAL, 1);
        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
        curl_setopt($ch, CURLOPT_TIMEOUT, 3);
        curl_setopt($ch, CURLOPT_USERAGENT, 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.146 Safari/537.36');
        $response = curl_exec($ch);
        curl_close($ch);
        
        $response = json_decode($response,true);
        
        if($response['ok'] !== true){
            return 0;
        }elseif($response['result']['status'] == 'left' || $response['result']['status'] == 'kicked'){
            return 1;//Users who left the supergroup or were kicked out
        }
        
        return 0;
    }
	
}
